ruby/IceDiscovery/greeter/README.md

```diff
@@ -0,0 +1,20 @@
+# IceDiscovery Greeter
+
+This demo illustrates how to use the IceDiscovery plugin with Ice for Ruby.
+
+Ice for Ruby supports only client-side applications. As a result, you first need to start a Greeter server implemented
```

Comment:
Should this say "an IceDiscovery Greeter server ..." a plain Greeter would not work with this client.